.github/workflows/InferenceSystem.yaml

```diff
@@ -223,6 +223,18 @@ jobs:
       run: |
         docker build . -t live-inference-system -f ./Dockerfile
 
+    - name: Test docker container
+      working-directory: InferenceSystem
+      run: |
+        docker run --rm \
+          --env-file .env \
+          -v $PWD/config:/config \
+          live-inference-system \
+          python3 -u ./src/LiveInferenceOrchestrator.py \
+          --config /config/Test/FastAI_LiveHLS_OrcasoundLab.yml \
```

Comment:
This test assumes that the orcasound hydrophone is working. It may be worth setting up a "test" instance that is guaranteed to be available.

Reply:
Filed issue #365 to track that idea.